src/Microsoft.IdentityModel.TestExtensions/TestTokenCreator.cs

```diff
@@ -0,0 +1,455 @@
+        /// </summary>
+        private const string _defaultTestIssuer = "http://Default.Issuer.com";
+
+        /// <summary>
+        /// The default test audience.
+        /// </summary>
+        private const string _defaultTestAudience = "http://Default.Audience.com";
+
+        /// <summary>
+        /// The default claim value for IssuedAt and NotBefore.
+        /// </summary>
+        private static DateTime _defaultIssuedAtNotBefore = DateTime.UtcNow - TimeSpan.FromHours(1);
+
+        /// <summary>
+        /// The default claim value for Expiration Time.
+        /// </summary>
+        private static DateTime _defaultExpirationTime = DateTime.UtcNow + TimeSpan.FromHours(23);
+
+        /// <summary>
+        /// The default set of claims for test tokens generated by this class.
+        /// </summary>
+        private static List<Claim> _payloadClaims = new List<Claim>()
+        {
+            new Claim(JwtRegisteredClaimNames.Email, "Bob@contoso.com", ClaimValueTypes.String, _defaultTestIssuer, _defaultTestIssuer),
+            new Claim(JwtRegisteredClaimNames.GivenName, "Bob", ClaimValueTypes.String, _defaultTestIssuer, _defaultTestIssuer),
+            new Claim(JwtRegisteredClaimNames.Iss, _defaultTestIssuer, ClaimValueTypes.String, _defaultTestIssuer, _defaultTestIssuer),
+            new Claim(JwtRegisteredClaimNames.Aud, _defaultTestAudience, ClaimValueTypes.String, _defaultTestIssuer, _defaultTestIssuer),
+            new Claim(JwtRegisteredClaimNames.Iat, EpochTime.GetIntDate(_defaultIssuedAtNotBefore).ToString(), ClaimValueTypes.String, _defaultTestIssuer, _defaultTestIssuer),
+            new Claim(JwtRegisteredClaimNames.Nbf, EpochTime.GetIntDate(_defaultIssuedAtNotBefore).ToString(), ClaimValueTypes.String, _defaultTestIssuer, _defaultTestIssuer),
+            new Claim(JwtRegisteredClaimNames.Exp, EpochTime.GetIntDate(_defaultExpirationTime).ToString(), ClaimValueTypes.String, _defaultTestIssuer, _defaultTestIssuer),
+        };
+
+        /// <summary>
+        /// A set of <see cref="RSAParameters"/> which should be different than what a caller provides as their SigningCredentials.
+        /// </summary>
+        private static RSAParameters _rsaParameters_4096 = new RSAParameters
+        {
+            D = Base64UrlEncoder.DecodeBytes("B15EJtB2qDEtE2z4k6dR-o_fQGoFtQR10FfMWt3XSE44jPYLMKyr3cc0NQkw7DsMpU8Olqo-enIX1BD9vVdWXAY-gR4yxJtRyiVcdAJiNHH6GN8s_GA21Z-6AZR_FTGgmJStEUe-KupSwzYdjo6f5S8dHyaJpLuRhhIx8mLExdhA7543Puwt-MMKj46rcEEKXhe2jNZl1vDgj8-71ZjTvE6HW__invbNX3ZefrZJCD-rDI9SyCkktq9msKfodFtyKfQ0-EYG6rxk7hpAwtilr418N-0I7oYh4OJHC44sRQswL7aCuxHB6qJbkPm7D5QIrrdd4ts5hYNoHd3YJIhr_H09VFnql5CfIyk2NbxHwbZAkGuA00ceBJ0RuIWvrxf-66ZbcNUz2U7l2GEF5qUsy7T4Zi6JtH7I3b4EaO620X-SZ5GXBWmxWBthVi2St3z7vAGl6Nv6F-pk7IjVZ4jj3EJGgCpHZEZObAD7y5IpslBiA4F24qitYfI4RPuncz78V2aKu8Zi6O19PFqQs1Ky3fEV5-fQjPqtu9944omHiwx0Jsjy0SU5MdM6indY75sTKM0xmOFCrF4poGFGJMHW-PJmFy2pCC9O39iz9ki32vJlvYo085a5vrVzKni_fY3HOJKC0NMMlwvAasedT2ZeN1hVVjSqlQwjGQTmDjcx07k"),
```

Comment:
are these okay to have here?

Reply:
Are you able to flush out your concern? These are fake and are already public in GitHub. Is there a place you feel they would be better placed?